openapi.yaml

```diff
@@ -10,7 +10,7 @@
       "name" : "Apache 2.0",
       "url" : "https://www.apache.org/licenses/LICENSE-2.0.html"
     },
-    "version" : "2.5.0"
+    "version" : "2.4.0"
```

Comment:
This is strange that it is moving it back to 2.4.0? can you check if there is anything a miss there?

everything else looks good!